.babelrc

```diff
@@ -1,11 +1,13 @@
 {
   "presets": [
-    ["eslatest-node6", { "loose": true }]
```

Comment:
.babelrc files are json files. Though it'll work, linters will show that there are errors, which might hide actual errors. Babel 7 will have .babelrc.js support, can't wait.